README.md

```diff
@@ -39,9 +482,9 @@ public class Stock
     public double CurrentPrice { get; }
 }
 ```
-### Collections
+### Supported Collections
```

Comment:
This may be me being overly focused on correct terminology, but I titled this section "Collections" because that's the Hydra term. I used `SupportedCollection` as a semantic for designating a collection for a Hydra Supported Class. I meant to clarify this in the README, but I obviously left it out.

I primarily want to avoid confusing people who may not be familiar with the Hydra spec.

Reply:
Yeah, I guess you're right. The spec seems to only define SupportedProperty and then it's just Class, Resource, and Collection. Happy to change it to the proper form.

Reply:
Or maybe "Collections of resources and the Hydra Collection"